### PR TITLE
System path with spaces fixes

### PR DESCRIPTION
**Pull request by ke2ulpatani**
Fixes for spaces in folder path

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const isWsl = require('is-wsl');
 const chalk = require('chalk');
 const child_process = require('child_process')
 const version = require('./package.json').version;
+__dirname = __dirname.replace(/ /g,"\\\ ");
 const virtcrudVersion = JSON.parse(child_process.execSync(`cd ${__dirname} && npm list virtcrud --json`).toString()).dependencies.virtcrud.resolved.split('#')[1].substring(0, 7);
```
Comment (chrisparnin) [inline index.js]:
```suggestion
 const virtcrudVersion = JSON.parse(child_process.execSync(`npm list virtcrud --json`, {cwd: __dirname}).toString()).dependencies.virtcrud.resolved.split('#')[1].substring(0, 7);

```

Comment (chrisparnin) [reply index.js]: A better fix would involve adjusting passing the cwd. Which should also escape spaces.

File: index.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ const isWsl = require('is-wsl');
 const chalk = require('chalk');
 const child_process = require('child_process')
 const version = require('./package.json').version;
+__dirname = __dirname.replace(/ /g,"\\\ ");
```
Comment (chrisparnin) [inline index.js]:
```suggestion
```
Remove this line.

**Comment by ssmirr** [comment]
Fixed in https://github.com/ottomatica/bakerx/commit/724ffbed008e198656cb5d15905cf37990d3a4d0. Thanks for the PR @ke2ulpatani!

**Pull request closed without merging**